kvm/v12.2.0/release.diff

```diff
@@ -0,0 +1,49 @@
+apiVersion: release.giantswarm.io/v1alpha1                         apiVersion: release.giantswarm.io/v1alpha1
+kind: Release                                                      kind: Release
+metadata:                                                          metadata:
+  creationTimestamp: null                                            creationTimestamp: null
+  name: v12.1.0                                                 |    name: v12.2.0
+spec:                                                              spec:
+  apps:                                                              apps:
+  - name: cert-exporter                                              - name: cert-exporter
+    version: 1.2.3                                                     version: 1.2.3
+  - name: chart-operator                                             - name: chart-operator
+    version: 0.13.0                                                    version: 0.13.0
+  - componentVersion: 1.6.5                                          - componentVersion: 1.6.5
+    name: coredns                                                      name: coredns
+    version: 1.2.0                                                     version: 1.2.0
+  - componentVersion: 1.9.7                                          - componentVersion: 1.9.7
+    name: kube-state-metrics                                           name: kube-state-metrics
+    version: 1.1.1                                                     version: 1.1.1
+  - componentVersion: 0.3.6                                          - componentVersion: 0.3.6
+    name: metrics-server                                               name: metrics-server
+    version: 1.1.1                                                     version: 1.1.1
+  - name: net-exporter                                               - name: net-exporter
+    version: 1.9.0                                                     version: 1.9.0
+  - componentVersion: 0.34.1                                    <
+    name: nginx-ingress-controller                              <
+    version: 1.8.1                                              <
+  - componentVersion: 1.0.1                                          - componentVersion: 1.0.1
+    name: node-exporter                                                name: node-exporter
+    version: 1.3.0                                                     version: 1.3.0
+  components:                                                        components:
+  - name: app-operator                                               - name: app-operator
+    version: 1.0.0                                                     version: 1.0.0
+  - name: calico                                                     - name: calico
+    version: 3.14.1                                                    version: 3.14.1
```

Comment:
@stone-z is the rule correct, requiring calico 3.15.1 for kvm, 3.14.1 not enough? Asking since the rule definition references https://github.com/giantswarm/giantswarm/issues/11307 which does not mention 3.15.1

Reply:
In [the request](https://github.com/giantswarm/releases/pull/377) we decided to baseline all the next 12.x releases on 3.15 so they're all the same and patched with regard to the IPAM issue and CVEs. Would be great to include here if it's not broken

Reply:
@stone-z clear, thanks for heads up.
Will give it a try, changing release definition only, hopefully no kvm-operator release is needed.
If it is needed, will need team ludacris help, or drop it.

Reply:
Changing calico version alone in release definition wasn't enough.
kvm-operator 3.12.1 is currently being used in the KVM 12.2.0 WIP platform release.

@giantswarm/team-ludacris is there kvm-operator release available that supports calico 3.15.1?

Reply:
Agreed with @stone-z @MartaD and @tfussell on Slack to remove calico from 12.2.0, see https://gigantic.slack.com/archives/C90BZMUJ1/p1596114714442100?thread_ts=1596096575.441100&cid=C90BZMUJ1